cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -351,6 +351,42 @@ select users:
   namespace. It is recommended that this property be set to the list of users that will
   administer and manage the CDAP installation.
 
+You can use the :ref:`CDAP CLI <cdap-cli>` to issue :ref:`security commands
+<cli-available-commands-security>`:
+
+- To grant a principal permission to perform certain actions on an entity, use::
```

Comment:
Use privilege? We already have actions and privileges, adding permissions might make it even more confusing.